lib/src/code_field/code_field.dart

```diff
@@ -198,7 +195,7 @@ class CodeFieldState extends State<CodeField> {
               child: Text(longestLine, style: textStyle),
             ), // Add extra padding
           ),
-          widget.expands ? Expanded(child: codeField) : codeField,
+          if (widget.expands) Expanded(child: codeField) else codeField,
```

Comment:
Please revert this for less diff with the original. This way it is easier to check if the original editor has any new features we need to grab to ourselves.